### PR TITLE
Add RHEL-08-010350 STIG to existing rule

### PR DESCRIPTION
**Pull request by carlosmmatos**
#### Description:
This change adds the RHEL8-08-010350 STIG ID to 2 existing rules that deal with shared library directories.
- Added/updated Ansible remediations in both rules
- Added bash remediation in directory rule
- Added RHEL8 references

### CODE REVIEW

File: linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <unix:file_object comment="system-wide directories" id="root_permissions_for_system_wide_library_files" version="1">
     <!-- Checks that system-wide library files in /lib, /lib64, /usr/lib, /usr/lib64
         are owned by root. -->
-    <unix:path operation="pattern match">^\/lib(64)?|^\/usr\/lib(64)?</unix:path >    
+    <unix:path operation="pattern match">^\/lib\/|^\/lib64\/|^\/usr\/lib\/|^\/usr\/lib64\/</unix:path>
```
[the remark this answers is not in the document]
Comment (cipherboy) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml:19]: Do you perhaps need a trailing `$` to indicate that the entire path must match? :-)

[the remark this answers is not in the document]
Comment (vojtapolasek) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml:19]: Seems @cipherboy is right, try this:
^\/lib(64)?$|^\/usr\/lib(64)?$
I think that trailing slash will not work, I don't think that the "path" element is expected to end with the slash. But it is not precisely documented afaik.

[the remark this answers is not in the document]
Comment (carlosmmatos) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml:19]: Will try in a sec

[the remark this answers is not in the document]
Comment (carlosmmatos) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml]: @cipherboy @vojtapolasek That worked.. so my follow-up question is - what about the other rules that follow this approach? Are we in agreement that we should update those as well?

[the remark this answers is not in the document]
Comment (vojtapolasek) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml]: I would say yes, let's unify it. It should not change their functionality, well, it should actually improve it.

[the remark this answers is not in the document]
Comment (cipherboy) [reply linux_os/guide/system/permissions/files/permissions_within_important_dirs/root_permissions_syslibrary_files/oval/shared.xml]: The other thing here is @dodys is eventually going to upstream some changes to `file_*` templates that will help here as well. I think he'll probably get them in next early week.

**Review by Mab879 — APPROVED** [review]
LGTM

**Pull request merged**